packages/expo-in-app-purchases/android/src/main/java/expo/modules/inapppurchases/InAppPurchasesModule.java

```diff
@@ -0,0 +1,96 @@
+package expo.modules.inapppurchases;
+
+import java.util.List;
+
+import android.content.Context;
+import android.app.Activity;
+
+import org.unimodules.core.ExportedModule;
+import org.unimodules.core.ModuleRegistry;
+import org.unimodules.core.Promise;
+import org.unimodules.core.interfaces.ExpoMethod;
+import org.unimodules.core.interfaces.ActivityProvider;
+import org.unimodules.core.interfaces.services.EventEmitter;
+import org.unimodules.core.interfaces.ModuleRegistryConsumer;
+
+public class InAppPurchasesModule extends ExportedModule implements ModuleRegistryConsumer {
+  private static final String TAG = InAppPurchasesModule.class.getSimpleName();
+  private static final String NAME = "ExpoInAppPurchases";
+
+  private BillingManager mBillingManager;
+  private ModuleRegistry mModuleRegistry;
+  protected static EventEmitter sEventEmitter;
```

Comment:
Make this an instance variable. Statics will cause problems when more than one module of the same type is instantiated unless they account for conflicts between instances.